bindings/node/lib/clientEncryption.js

```diff
@@ -363,6 +366,16 @@ module.exports = function (modules) {
         });
       });
     }
+
+    /**
+     * Ask the user for KMS credentials.
+     */
+    async askForKMSCredentials() {
+      /* This returns anything that looks like the kmsProviders original input
+       * option. It can be empty, and any provider specified here will override
+       * the original ones. */
```

Comment:
I think this comment is helpful as docs, you could move it to the function's jsdoc

Reply:
Sure, done! :+1: